nml/actions/action0properties.py

```diff
@@ -255,7 +255,7 @@ def cargo_list(value, max_num_cargos):
 }
 
 def ottd_display_speed(value, divisor, unit):
-    return int(value.value / divisor) * 10 // 16 * unit.ottd_mul >> unit.ottd_shift
+    return (round(value.value // divisor * 10 * unit.ottd_mul) >> unit.ottd_shift) // 16
```

Comment:
Why is there an integer division "// divisor" inside a round? That looks wrong to me.

Reply:
I think this function is meant to replicate the exact behaviour of OTTD's ConvertKmhishSpeedToDisplaySpeed.
Thus I would expect:
`(round(value.value / divisor * 10 * unit.ottd_mul) >> unit.ottd_shift) // 16`

Reply:
I was actually wondering whether we may have a round there at all... or whether we get some platform dependency there which we might not want.
Though without an integer being left of the shift operator, we get a type error...